tests/features/wi-elections.feature

```diff
@@ -89,10 +89,6 @@ Feature: WI Elections
     | candidate                             | county    | office                                    | district  | ward                                          | votes | total |
     | Ken Sortedahl                         | St. Croix | St Croix County Circuit Court, Branch 4   |           | VILLAGE OF DEER PARK                          | 3     | 50    |
 
-  Examples: 20080219__wi__primary__ward.csv
```

Comment:
Why is this test deleted?

Reply:
Not sure what happened there. I've added it back in